web/beacon-app/src/features/projects/components/ProjectSetup.tsx

```diff
@@ -21,7 +21,7 @@ const ProjectSetup = ({ config, warningMessage }: ProjectSetupProps) => {
         data-testid="project-setup"
       >
         <Card.Header>
-          <Heading as="h3" className="pb-2 text-warning-500">
+          <Heading as="h3" className="pb-2 font-bold">
```

Comment:
The color was removed to improve color contrast and make the text easier for user's to read.